docs/configuration/integrations/gitlab-cloud.md

```diff
@@ -1,35 +1,55 @@
 # GitLab Cloud
 
-## GitLab Application
-
 Follow the instructions below to set up the Codacy integration with GitLab Cloud:
 
-1.  Create a [new application](https://gitlab.com/profile/applications) pointing to your local Codacy deployment URL with `api`, `read user` and `read repository` scopes.
+1.  Open <https://gitlab.com/profile/applications>.
```

Comment:
Will this be formatted as a link? Otherwise just convert it to one, it's simpler.

Reply:
Yes, this is a bare URL in Markdown and is rendered like this:

![image](https://user-images.githubusercontent.com/60105800/80391096-321d8c00-88a5-11ea-910b-774ab5bdfa65.png)

(For the self-hosted providers I'm not formatting these URLs as anchor links because the user must enter the correct hostname of their instance, so clicking the link would only be confusing.)